From edbcb0b73c831f2ae47f1a28d05d541c4abaabdd Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Tue, 5 Oct 2021 14:43:37 -0700
Subject: [PATCH] kms/device: Add virtio_gpu to deny list

virtio_gpu is used in virtual machine environments, meaning they need
accurate cursor hotspots, which currently needs non-atomic mode setting.

This is the equivalent of 9179d4a on the main branch, but the list
is in a different file on the GNOME 40 branch.

Related: https://bugzilla.redhat.com/show_bug.cgi?id=2009304

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 src/backends/native/meta-kms-device.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/backends/native/meta-kms-device.c b/src/backends/native/meta-kms-device.c
index c388096d59..85aded9a6f 100644
--- a/src/backends/native/meta-kms-device.c
+++ b/src/backends/native/meta-kms-device.c
@@ -252,6 +252,7 @@ is_atomic_allowed (const char *driver_name)
     "vboxvideo",
     "nvidia-drm",
     "tegra",
+    "virtio_gpu",
     NULL,
   };
 
-- 
2.32.0

